### ISSUE TITLE
The calculation of weighted accurary

### ISSUE THREAD

**Issue by qhhuang-ai** [amirhfarzaneh/dacl#2]
Thanks for your contribution to the community! As shown in  paperwithcode.com, the unweighted accuracy is calculated by (#corrected predictions)/(#total instances), about 80.44% in your paper. But the calculation method of accuracy or weighted accuracy  (about 87.78% in your paper) seems  not clear in the paper or in the codes. The weighted vector for the weighted accuracy is also not clear in the other related papers.  I thought it was calculated by taking average over the accuracy of each expression class, but the accuracy obtained from such a calcuation is often smaller than the unweighted accuracy in the experiments.  Could you please tell us how the weighted vector for  the weighted accuracy  is calculated in the expriment? Looking forward to your reply. Thanks again!

**Comment by izhimahu** [comment]
Do you have a pre-training model?Because I made an error `No such file or directory: '/users/amirhf/models/resnet18_msceleb.pth'`

**Comment by qhhuang-ai** [comment]
> Do you have a pre-training model?Because I made an error `No such file or directory: '/users/amirhf/models/resnet18_msceleb.pth'`

The authors haven't released the pre-trained model. Just disable the codes for pre-trained settings, and train the model from a scratch.

**Comment by izhimahu** [comment]
Thank you!

**Comment by qhhuang-ai** [comment]
> Thank you!

You're welcome.

**Comment by zhanghee** [comment]
I have the same question. Looking forward to the author's reply. Thank you!

**Comment by wbb95** [comment]
Sorry to interrupt: I just trained sparsecenterloss on my own model, but the result seems to be reduced on RAF-DB from 84.8% to 84.3%. Have you or others got the improved result

**Comment by qhhuang-ai** [comment]
> Sorry to interrupt: I just trained sparsecenterloss on my own model, but the result seems to be reduced on RAF-DB from 84.8% to 84.3%. Have you or others got the improved result

Did you pretrain your model on a large-scale facial dataset e.g. MS1M-Celeb or VggFace2? I also can not reproduce the results so far.

**Comment by wbb95** [comment]
i haven't done this maybe helpful.
